src/main/java/org/folio/rest/impl/AbstractHelper.java

```diff
@@ -93,8 +93,9 @@ protected Void handleError(Throwable throwable) {
     final Throwable t = throwable.getCause();
     final String message = t.getMessage();
     if (t instanceof HttpException) {
-      final int code = ((HttpException) t).getCode();
-      result = succeededFuture(buildErrorResponse(code, new Error().withMessage(message)));
+      final int httpCode = ((HttpException) t).getCode();
+      final String errorCode = ((HttpException) t).getErrorCode();
+      result = succeededFuture(buildErrorResponse(httpCode, new Error().withMessage(message).withCode(errorCode)));
     } else if (t instanceof ValidationException) {
```

Comment:
It seems that now it is possible to replace the `ValidationException `with `HttpException`.